docker/Dockerfile.ubuntu22

```diff
@@ -0,0 +1,87 @@
+FROM ubuntu:22.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+ARG TAG_VERSION=main
+
+RUN apt-get update && apt-get install -y \
```

Comment:
liblog4cplus-1.1-9 and   libssl1.1 not required?

Reply:
liblog4cplus-1.1-9 - > liblog4cplus-dev, libssl1.1 -> libssl-dev